app/models/recruitment_cycle.rb

```diff
@@ -5,11 +5,15 @@ module RecruitmentCycle
   }.freeze
 
   def self.current_year
-    if Time.zone.today < EndOfCycleTimetable.find_reopens
-      2020
-    else
-      2021
+    cycle = EndOfCycleTimetable::CYCLE_DATES.detect do |year, dates|
```

Comment:
Maye push this implementation into EndOfCycleTimetable so that it remains the only thing that knows when cycles start and end?

Reply:
Yep that makes sense having this logic in one class only